sale_order_type/models/sale_order_type.py

```diff
@@ -53,7 +53,7 @@ class SaleOrderTypology(models.Model):
         check_company=True,
     )
     analytic_account_id = fields.Many2one(
-        comodel_name="account.analytic.account",
```

Comment:
This kind of change is not really necessary. Explicit arguments is better

Reply:
Corrected!